packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -354,7 +354,7 @@ protected function askForDatabaseDetails()
 
         foreach ($databaseDetails as $key => $value) {
             if ($value) {
-                $this->envUpdate($key, $value);
+                $this->envUpdate($key, $value, true);
```

Comment:
@tusharnain4578 ,

I think we don't need these changes because when we try to enter a password invalid or with a special character, the MySQL connection will not be created and it will always give an error to us.

Reply:
@amit-webkul The special characters are allowed in mysql secure passwords, and in most of the sql server generated a strong password with special characters (if auto generated) which includes '#' also, which is causing issues in .env file as if string containing '#' is not enlosed in quotes, the text after it will be treated as comment.